site/content/in-dev/unreleased/managing-security/external-pdp/opa.md

```diff
@@ -0,0 +1,363 @@
+---
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+title: Open Policy Agent (OPA) Integration
+linkTitle: OPA
+type: docs
+weight: 100
+---
+
+{{% alert title="Experimental Feature" color="warning" %}}
+**OPA integration is currently an experimental feature** and may undergo breaking changes in future versions. Use with caution in production environments.
+{{% /alert %}}
+
+This page describes how to integrate Apache Polaris (Incubating) with [Open Policy Agent (OPA)](https://www.openpolicyagent.org/) for external authorization.
+
+## Overview
+
+Open Policy Agent (OPA) is a general-purpose policy engine that enables unified, context-aware policy enforcement across your stack. OPA provides a high-level declarative language (Rego) for authoring policies and APIs to offload policy decision-making from your software.
+
+Key benefits of using OPA with Polaris:
+
+- **Flexible policy language**: Write authorization logic in Rego, a powerful declarative language
+- **Centralized policy management**: Manage all policies in a single location
+- **Policy testing**: Write unit tests for your authorization policies
+- **Rich ecosystem**: Integrate with policy bundles, decision logs, and management tools
+- **Attribute-based access control**: Make decisions based on user attributes, resource properties, and environmental context
+
+## Prerequisites
+
+Before configuring OPA integration:
+
+1. **OPA Server**: Deploy and configure an OPA server accessible from Polaris
+2. **Policy Definition**: Write and deploy authorization policies to OPA
+3. **Network Access**: Ensure Polaris can reach the OPA server
+
+## Quick Start
+
+### 1. Deploy OPA
+
+Deploy OPA server with your policies. For example, using Docker:
+
+```bash
+docker run -d \
+  --name opa \
+  -p 8181:8181 \
+  -v $(pwd)/policies:/policies \
+  openpolicyagent/opa:latest \
+  run --server --addr :8181 /policies
+```
+
+### 2. Create a Policy
+
+Create a policy file (e.g., `policies/polaris.rego`):
+
+```rego
```

Comment:
Too sad - guess we need to bump the Hugo version to get syntax highlighting for Rego.